app/containers/Main/actions.js

```diff
@@ -106,10 +106,15 @@ export function login() {
           console.log(error, 'error caught');
         }
         dispatch(updateIsLogIn(true));
-      } else if (!response.data.meta.success && response.data.meta.message === "user is not registered") {
-        Actions.registerEmail()
-      }
-      dispatch(updateIsFetching(false));
+      } else if (!response.data.meta.success && response.data.meta.message === "username not found") {
```

Comment:
try checking out new response from backend, you should see some flag that show user is already registered. use that flag now.